docs/rococo.md

```diff
@@ -33,13 +33,15 @@ cargo build --release --bin interbtc-parachain --features rococo-native
 sudo cp ./target/release/interbtc-parachain /usr/local/bin/
 ```
 
-Run polkadot-launch with [this config](./xcm-config.json).
+Run polkadot-launch. We provide two configations: one to start a [single parachain](./rococo-local-single-chain.json), and one that sets up [two parachains](rococo-local-xcm-setup.json) that can be used for testing xcm communication.
```

Comment:
```suggestion
Run polkadot-launch. We provide two configurations: one to start a [single parachain](./rococo-local.json), and one that sets up [two parachains](rococo-local-xcm.json) that can be used for testing xcm communication.
```

Reply:
@sander2 these now do not match the names of the configuration files. Feel free to revert this actually or rename the files more succinctly.